server/atelier.js

```diff
@@ -102,11 +102,31 @@ const reportAnswer = (answerId) => {
     });
 };
 
+const getQuestions = (id) => {
```

Comment:
This is almost the same as the function I wrote, just added count 50 so I could get as much questions as possible